docs/conf.py

```diff
@@ -122,7 +122,10 @@
     'docroles'
 ]
 
-autodoc_default_flags = ['show-inheritance', 'members']
+autodoc_default_options = {
+    'show-inheritance': True,
+    'members': True
+}
 
 viewcode_import = True
```

Comment:
This line is what I was talking about.

Reply:
Thanks @XD-DENG I added this fix as well.

Reply:
Thanks!

Will merge once CI passes.